packages/jaeger-ui-components/src/TraceTimelineViewer/VirtualizedTraceView.test.tsx

```diff
@@ -116,17 +126,27 @@ describe('<VirtualizedTraceViewImpl>', () => {
   });
 
   it('sets the trace for global state.traceTimeline', () => {
-    expect(props.setTrace.mock.calls).toEqual([[trace, props.uiFind]]);
-    props.setTrace.mockReset();
+    expect((props.setTrace as jest.Mock).mock.calls).toEqual([[trace, props.uiFind]]);
+    (props.setTrace as jest.Mock).mockReset();
```

Comment:
instead of `as jest.Mock` you should be able to do `jest.mocked(props.setTrace)`, FYI.

Reply:
It's up to you. They're functionally identical, but _personally_ I believe it's worth, in new code, to limit our use of problematic patterns (`as` type assertions) to set the right example for others.